app/ui/legacy/src/main/java/com/fsck/k9/activity/MessageListActivityAppearance.kt

```diff
@@ -0,0 +1,67 @@
+package com.fsck.k9.activity
+
+import com.fsck.k9.Account
+import com.fsck.k9.FontSizes
+import com.fsck.k9.K9
+
+class MessageListActivityAppearance(accounts: List<Account>) {
```

Comment:
I suggest changing this to

```kotlin
data class MessageListActivityAppearance(
    val appTheme: K9.AppTheme,
    val language: String,
    // …
    val fontSizeMessageListSubject: Int,
    // …
) {
    companion object {
        @JvmStatic
        fun create() = MessageListActivityAppearance(
            appTheme = K9.appTheme,
            language = K9.k9Language,
            // …
            fontSizeMessageListSubject = K9.fontSizes.messageListSubject,
            // …
        )
}
```

By using a data class you don't need to write the `equals()` method.

The creation could be done with default values for constructor parameters. But I like having a separate static factory method better.

Reply:
Thank you for the review!